.env.development

```diff
@@ -43,6 +43,7 @@ FEATURE_SETTINGS_PAGE_APPEARANCE_TAB='true'
 FEATURE_LEARNER_CREDIT_MANAGEMENT='true'
 FEATURE_CONTENT_HIGHLIGHTS='true'
 FEATURE_API_CREDENTIALS_TAB='true'
+FEATURE_DEMO_DATA='false'
```

Comment:
[inform] It doesn't matter whether you make it true or false because the code takes this variable as a string, not a boolean variable. if you want to disable this flag on your development environment, you must remove this variable from here.